packages/portalnetwork/test/wire/utp.spec.ts

```diff
@@ -140,3 +197,40 @@ tape('uTP encoding tests', (t) => {
     st.end()
   })
 })
+
+tape('uTP packet handling', async (t) => {
```

Comment:
I think this whole test should already be covered by our integration test that is intended test uTP.  I ran that test locally and it looks like it's broken right now will need to look at that.